rmgpy/species.py

```diff
@@ -403,7 +403,14 @@ def getSymmetryNumber(self):
         if self.symmetryNumber < 1:
             cython.declare(resonanceHybrid = Molecule, maxSymmetryNum = cython.short)
             resonanceHybrid = self.getResonanceHybrid()
-            self.symmetryNumber = resonanceHybrid.getSymmetryNumber()
+            try:
+                self.symmetryNumber = resonanceHybrid.getSymmetryNumber()
+            except KeyError:
+                logging.warning('Wrong bond order generated by resonance hybrid.')
+                logging.warning('Resonance Hybrid: {}'.format(resonanceHybrid.toAdjacencyList()))
+                for index, mol in enumerate(self.molecule):
+                    logging.warning("Resonance Structure {}: {}".format(index,mol.toAdjacencyList()))
+                raise
```

Comment:
I guess this line is just for your debugging purpose. Maybe add exception with helpful information.